README.md

```diff
@@ -831,6 +831,7 @@ Instructs the module to ignore a specified rule ID. Note that ignoring a rule in
 	location / {
 		access_by_lua '
 			waf:set_option("ignore_rule", 40294)
+			waf:set_option("ignore_rule", {40002, 41036})
```

Comment:
It is probably a bad spot, maybe somewhere earlier a mention of the Lua table format would be good? If you don't know Lua at all it would be helpful.